CMakeLists.txt

```diff
@@ -184,7 +184,8 @@ set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
 
 if(MSVC)
   if(QT_PORT)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP /GF /GR /DWIN32 /D_CONSOLE /W3 /EHsc /Zi")# Add MSVS 2010 for Qt specific flags
+     # Add MSVS 2010 for Qt specific flags
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP /GF /GR /D_CRT_SECURE_NO_WARNINGS /D_CONSOLE /W3 /EHsc /wd4005 /wd4996 /nologo")
```

Comment:
/wd4005 /wd4996 - to suppress the warnings in step of code generation
/nologo - https://msdn.microsoft.com/en-us/library/ef4d60dk(v=vs.100).aspx